scp.c

```diff
@@ -1482,7 +1483,7 @@ sink_sftp(int argc, char *dst, const char *src, struct sftp_conn *conn)
 		debug("Fetching %s to %s\n", g.gl_pathv[i], abs_dst);
 		if (globpath_is_dir(g.gl_pathv[i]) && iamrecursive) {
 			if (download_dir(conn, g.gl_pathv[i], abs_dst, NULL,
-			    pflag, 1, 0, 0) == -1)
+			    pflag, SFTP_PROGRESS_ONLY, 0, 0) == -1)
```

Comment:
perhaps set this depending on the verbose flag? could be useful for debug.

Reply:
verbose already gives a lot more useful stuff since I hooked up log_init() in main :)